pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -705,16 +787,20 @@ class Build {
 
         context.node('master') {
             context.stage("sign installer") {
-                if (buildConfig.TARGET_OS == "windows") {
-                    try {
+                try {
+                    if (buildConfig.TARGET_OS == "windows") {
                         signInstallerJob(versionData);
                         context.sh 'cd workspace/target/ && for file in $(ls *.tar.gz *.pkg *.msi); do sha256sum "$file" > $file.sha256.txt ; done'
                         writeMetadata(versionData, false)
                         context.archiveArtifacts artifacts: "workspace/target/*"
-                    } catch (e) {
-                        context.println("Failed to build ${buildConfig.TARGET_OS} installer ${e}")
-                        currentBuild.result = 'FAILURE'
+
+                    } else if (buildConfig.TARGET_OS == "aix") {
```

Comment:
Should we add a comment here that aix sign tool is not supported on aix that's why we have to sign separately?